### ISSUE TITLE
[Feature request] Shared collections

### ISSUE THREAD

**Issue by charlyhue** [benjaminjonard/koillection#316]
Hello,

Would it be possible to add the possibility of sharing a collection between several authenticated users?

Thank you

**Comment by benjaminjonard** [comment]
Hello, 

Yes it seems like a good idea. Right now there is a Public or Private status, would a third option like "Shared only with authenticated users" be enough for now ? 
I may add on option later to pick users individually but it will require more work and I won't have time to work on this right now.

**Comment by charlyhue** [comment]
Yes, shared only with authenticated users is what I need. Pick users individually should be a must have in the future.

**Comment by Gabbalo** [comment]
I'd support this suggestion, but concerning the age of it, there might be a very low chance to actually get it, am I right? :/

**Comment by LargoUsagi** [comment]
Just adding that shared collections would be useful, especially when managing collections of items that are shared in a family like physical media.
